README.md

```diff
@@ -50,12 +50,12 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
 Your Answer:
 
 ```text
-___: Defines the distance between an element's border and adjacent elements' borders.
-___: Inserts a "wall" around an element.
-___: Defines the distance between an element's content and its border.
-___: Defines the width of an element.
-___: Overlays a "wall" on top of an element.
-___: Defines the distance between the center of an element and the center of the adjacent element.
+____: Defines the distance between an element's border and adjacent elements' borders.
+outline: Inserts a "wall" around an element.
+padding: Defines the distance between an element's content and its border.
+margin: Defines the width of an element.
+border: Overlays a "wall" on top of an element.
+_____: Defines the distance between the center of an element and the center of the adjacent element.
 ```
```

Comment:
The correct answers are...

```
margin: Defines the distance between an element's border and adjacent elements' borders.
border: Inserts a "wall" around an element.
padding: Defines the distance between an element's content and its border.
___: Defines the width of an element.
outline: Overlays a "wall" on top of an element.
___: Defines the distance between the center of an element and the center of the adjacent element.
```

Thinking about it visually helps me: https://css-tricks.com/the-css-box-model/